test/alternator/test_describe_table.py

```diff
@@ -112,12 +120,73 @@ def test_describe_table_item_count(test_table):
     got = test_table.meta.client.describe_table(TableName=test_table.name)['Table']
     assert 'ItemCount' in got
 
-# Similar test for estimated size in bytes - TableSizeBytes - which again,
-# may reflect the size as long as six hours ago.
-def test_describe_table_size(test_table):
-    got = test_table.meta.client.describe_table(TableName=test_table.name)['Table']
-    assert 'TableSizeBytes' in got
-    assert got['TableSizeBytes'] >= 0
+def test_describe_table_size_with_timeout_set_to_0(cql, dynamodb):
+    # UPDATE system.config SET value = 0 WHERE name = alternator_describe_table_info_timeout_in_seconds
```

Comment:
This comment is misplaced, it descibes the code a few lines below, config_value_context (and I'm not even sure it's helpful, I think the config_value_context is easier to understand).